packages/aws-cdk-lib/aws-apigateway/lib/deployment.ts

```diff
@@ -26,6 +26,13 @@ export interface DeploymentProps {
    * @default false
    */
   readonly retainDeployments?: boolean;
+
+  /**
+   * The stage of the API Gateway deployment.
+   *
+   * @default - If `stageName` is not specified, a "prod" stage will be automatically created for the API Gateway deployment.
```

Comment:
qq: Does Deployment creates the `stage` or only refer the existing stage? 
Because default stage `prod` is created by `RestApi` I think. Worth to double check. If so we can change the description something like 
"...default existing `prod` stage will be used for API Gateway Deployment"

Reply:
from what I understand Deployment does not create a new stage and only refers to the existing stage by `stageName` - I'll update the description